application/common/mail/layouts/html.php

```diff
@@ -1,34 +1,33 @@
 <?php
+
 use Sil\PhpEnv\Env;
 
+$brandColor = Env::get('EMAIL_BRAND_COLOR', '');
+$logo = Env::get('EMAIL_BRAND_LOGO', '');
+$maxWidth = Env::get('EMAIL_MAX_WIDTH', '600px');
+/* @var $content string email contents */
+
 ?>
 <!DOCTYPE html>
-<html>
+<html lang="eng">
```

Comment:
I _think_ this should be `en` or `en-US`, rather than `eng`

Reply:
Good catch, thank you.